docker/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+name: cwydsa
+services:
+  web:
+    image: fruoccopublic.azurecr.io/rag-webapp
+    ports:
+      - "8080:80"
+    env_file:
+      - path: ../.env
+        required: false
+      - path: $AZD_ENV_FILE
+        required: false
+  admin:
+    image: fruoccopublic.azurecr.io/rag-adminwebapp
+    ports:
+      - "8081:80"
+    environment:
+      - BACKEND_URL=http://backend
+      - FUNCTION_KEY=some-test-key
+    env_file:
+      - path: ../.env
+        required: false
+      - path: $AZD_ENV_FILE
+        required: false
+  backend:
+    image: fruoccopublic.azurecr.io/rag-backend
+    ports:
+      - "8082:80"
+    environment:
+      - AzureWebJobsSecretStorageType=files
+    env_file:
+      - path: .env
+        required: false
+      - path: $AZD_ENV_FILE
+        required: false
+    volumes:
+      -  ${HOST_DOCKER_FOLDER:-.}/function-host.json:/azure-functions-host/Secrets/host.json
```

Comment:
![snazzy](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZmt6ZGZ4MXl2d3dqcHV2YWFzam82bW5kN3N2enR3dWZnbWp2Y2d0eCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/oXg4pP0MAfLmOZk72Z/giphy.gif)

Reply:
This took me a while to figure out 😅 One of the downsides of dev containers!